packages/react-storage/src/components/StorageBrowser/Views/LocationActionView/LocationActionView.tsx

```diff
@@ -34,7 +35,12 @@ export const LocationActionView: LocationActionView = () => {
       >
         Cancel
       </button>
-      <LocationActionViewControls />
+      {actionType === 'CREATE_FOLDER' ? (
+        <CreateFolderActionViewControls />
```

Comment:
At the moment, this isn't styled or anything. Was thinking of wrapping the controls grid around the ui in `LocationActionView`. I think this could be a follow up, but at the moment create folder looks like this:

![image](https://github.com/user-attachments/assets/14cc4bbc-ca73-4413-a8a7-2a8174a3f33b)